src/typedefs.jl

```diff
@@ -2,9 +2,10 @@ const DefaultRefType = UInt32
 
 ## Pools
 
-# V is always set to CategoricalValue{T}
-# This workaround is needed since this type not defined yet
-# See JuliaLang/julia#269
+# Type params:
+# * `T` type of categorized values
+# * `R` integer type for referencing category levels
+# * `V` categorical value type that implements "categorical value" trait
```

Comment:
Repeating my comment from above: Could you remove quotes around mentions of "categorical value"? I don't find it very useful, and for example in the docstring below they are not present.